src/formio/components/Fieldset.stories.mdx

```diff
@@ -0,0 +1,146 @@
+import {Canvas, Meta, Story, ArgsTable} from '@storybook/addon-docs';
+import {FormioComponent, FormDecorator} from './story-util';
+
+<Meta
+  title="Form.io components/Fieldset"
+  argTypes={{
+    components: {
+      type: {name: 'array'},
+    },
+    id: {
+      type: {name: 'string'},
+    },
+    label: {
+      type: {name: 'string', required: false},
+    },
+    descriptionFamilyName: {
+      type: {name: 'string'},
+    },
+    descriptionGivenName: {
+      type: {name: 'string'},
+    },
+    labelFirstName: {
+      type: {name: 'string'},
+    },
+    labelLastName: {
+      type: {name: 'string'},
+    },
+    labelPrivacy: {
+      type: {name: 'string'},
+    },
+    placeholderFirstName: {
+      type: {name: 'string'},
+    },
+    placeholderLastName: {
+      type: {name: 'string'},
+    },
+  }}
+  decorators={[FormDecorator]}
+  parameters={{
+    controls: {sort: 'requiredFirst'},
+  }}
+/>
+
+# Field Set
+
+A Field Set can be used to create a title of an area of the form. This is useful to put inside
+Layout components or in between lots of related components. The Field Set is for display only and
+will not be saved to the API,
+
+Additionally the use of a field set increases accessibility: grouping related form controls makes forms more understandable for all users, as related controls are easier to identify, makign it easier for people to focus on smaller and more manageable groups.
+
+**Legend**
+
+A Field Set label that will appear as a Legend for the component in the form.
```

Comment:
there should be an arg/control for `hideHeader`, default is `false` (so it is visible).